src/main/xerus/monstercat/tabs/TabCatalog.kt

```diff
@@ -55,12 +60,69 @@ class TabCatalog: TableTab() {
 		table.visibleLeafColumns.addListener(ListChangeListener {
 			it.next(); Settings.VISIBLECATALOGCOLUMNS.putMulti(*it.addedSubList.map { it.text }.toTypedArray())
 		})
+
+		fun playTracks(add: Boolean){
+			val selected = table.selectionModel.selectedItems ?: return
+			GlobalScope.launch {
```

Comment:
Why are you starting coroutines everywhere?

Reply:
Same; this belongs to #15 . Please don't mess up the PRs